agent/ansible/Makefile

```diff
@@ -14,10 +13,12 @@ colldir = collections/ansible_collections/${namespace}/${collection}
 .DEFAULT_GOAL := test
 
 lint:
-	make -C pbench/agent
+	ansible-lint -c ansible-lint.yml
```

Comment:
The`ansible-lint.yml` file needs the addition

```
  - package-latest  # Package installs should not use latest.
```
to stop complaining about `latest`.